src/main/java/edu/umich/its/cpm/EmailFormatter.java

```diff
@@ -157,5 +270,11 @@ public String subStringFrom(String emailText, String contentToChopFrom) {
         return emailText.substring(posA);
     }
 
+    public String subStringInBetween(String emailText, String chopper) {
+        String choppingFirstBoundary = emailText.substring(chopper.length());
```

Comment:
Is this specific to emailText?  Why does the first line make sense?  Is this a clear name for the method?